src/components/organisms/EventCard/EventCard.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import ConferencePlaceHolder from '../../../assets/images/ConferencePlaceHolder.jpg'
+import {
+  EventCardStyled,
+  EventImage,
+  EventTitle,
+  EventDescription,
+  EventFooter,
+  EventAttendeeCounter,
+  EventDate
+} from './EventCard.styes'
+
+export function EventCard({
+  url,
```

Comment:
Please change the `url` name for something more explicit, like `imageUrl` or `imageSrc`

Reply:
Pablo Done!
I use imageUrl.